com/williamfiset/datastructures/binarysearchtree/SplayTree.java

```diff
@@ -0,0 +1,274 @@
+import com.williamfiset.datastructures.utils.TreePrinter;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
```

Comment:
Can you document in a few sentences what a splay tree it, what it's good for, maybe a little example, time complexity and basic things? Also feel free to give yourself credit as the author :)

Reply:
Oke, but should i keep them in the slides folder ?

Reply:
I just meant at the top of this file in a comment is fine.in the